public/doc/openapi-3-v3.0.json

```diff
@@ -11307,6 +11307,10 @@
             "readOnly": true,
             "type": "string"
           },
+          "forwardable_headers": {
```

Comment:
I do not think `forwardable_headers` should be exposed through the API.  We only need it for internal processing.  cc @syncrou @mkanoor

Reply:
I agree.  Seems we shouldn't expose forwardable headers in the openapi spec

Reply:
Yes its internal and should not be exposed.

Reply:
I can remove it from this PR, but generally the `openapi:generate` rake task must be extended to support skipping

Reply:
TPINVTRY-932